### PR TITLE
Update tag and push commands

### PR DESCRIPTION
**Pull request by jigar1101**
#### Pre-Submission Checklist
- [ ] Your pull request targets the `develop` branch.
- [ ] Branch starts with either `fix/` or `feature/`
- [ ] All new and existing tests pass the command `npm test`.
- [ ] Any new files are included in the grunt tasks

#### Type of Change
- [ ] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)

#### Checklist:
- [ ] Tested changes locally.
- [ ] New tests added?
- [ ] Closes currently open issue (replace XXXX with an issue no): Closes #XXXX

#### Full Description of your Changes

Extension of https://github.com/esfiddle/esfiddle/pull/281

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -72,9 +69,13 @@ jobs:
 
           # Use Docker `latest` tag convention
           [ "$VERSION" == "develop" ] && VERSION=latest
+          
+          #Date Time
+          $DATETIME=$(date +%s)
 
           echo IMAGE_ID=$IMAGE_ID
           echo VERSION=$VERSION
```
Comment (joshghent) [inline .github/workflows/deploy.yml:77]: Is this variable used anymore?

File: .github/workflows/deploy.yml [hunk 2]
```diff
@@ -72,9 +69,13 @@ jobs:
...
+          echo DATETIME=$DATETIME
 
-          docker tag $IMAGE_NAME $IMAGE_ID:$VERSION
-          docker push $IMAGE_ID:$VERSION
+          docker build . --file Dockerfile --tag $IMAGE_ID:$DATETIME --tag $IMAGE_ID:$DATETIME
+          docker push $IMAGE_ID
```
Comment (joshghent) [inline .github/workflows/deploy.yml:81]: For the master branch, can we please push a `latest` tag as well please

**Review by joshghent — COMMENTED** [review]
Nice work, couple of minor changes to make 👍

File: .github/workflows/deploy.yml (hunk 1 above)
Comment (jigar1101) [reply .github/workflows/deploy.yml:77]: My bad I pushed the same tag twice.

File: .github/workflows/deploy.yml (hunk 2 above)
Comment (jigar1101) [reply .github/workflows/deploy.yml:81]: My bad I pushed the same tag twice. I have updated the tag

**Review by joshghent — APPROVED** [review]
Nice work mate

**Pull request merged**